CHANGELOG.md

```diff
@@ -2,7 +2,33 @@
 
 ## Unreleased
 
-Unreleased: https://github.com/salemove/ios-sdk/compare/0.29.4...master
+Unreleased: https://github.com/salemove/ios-sdk/compare/0.30.0...master
+
+## [0.30.0] - 2021-07-26
+
+### Added
+
+- Added push notification support
+- Added possibility start engagement with media direction
+- Added possibility to fetch visitor information
+
+### Fixed
+
+- Fixed media streaming on engagement transfer
+
+[0.30.0]: https://github.com/salemove/ios-sdk/compare/0.29.6...0.30.0
+
+## [0.29.6] - 2021-07-01
+
+### Added
+
+- Added possibility to start engegment with media
```

Comment:
The support for this is incomplete in 0.29.6, so I think this has to be in 0.30.0, replacing the line about media direction.

Reply:
fixed 21356ced6c2c62b1ff9bbe22f18afb2577e4cfb8